.github/workflows/CI.yml

```
@@ -38,8 +38,14 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
-      - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
+      - uses: julia-actions/julia-buildpkg@latest
```

Please revert this change, in particular now that dependabot is running we should only use specific releases (see also #93):
```suggestion
      - uses: julia-actions/julia-buildpkg@v1
```